### ISSUE TITLE
How to use with Next.js

### ISSUE THREAD

**Issue by adueck** [froala/react-froala-wysiwyg#110]
I am having trouble trying to get Froala to work with [Next.js](https://nextjs.org/).

It would be great to have an example of how to use it. How would you be able to import all the necessary files in a way that works with [Next.js](https://nextjs.org/)?

**Comment by eli0shin** [comment]
I have found that the solution to Wysiwygs Next.js is to use dynamic import() wrapped inside a <SkipServerRender> Component so they are not loaded during SSR.

**Comment by adueck** [comment]
Yes, @vkhupenia provided me with a working solution in [this response](https://github.com/froala/react-froala-wysiwyg/issues/89#issuecomment-429782077) to #89.  

The only difficult thing was also getting JQuery working nicely with Next.js, which I did with the following in my `next.config.js`

```
//next.config.js

const webpack = require('webpack')

module.exports = {
    webpack: (config, { dev }) => {
        config.plugins.push(
            new webpack.ProvidePlugin({
                '$': 'jquery',
                'jQuery': 'jquery',
            })
        )
        return config
    }
}
```
